### ISSUE TITLE
Validation fails for subjects with labels with initial uppercase

### ISSUE THREAD

**Issue by effigies** [bids-standard/legacy-validator#873]
From [Neurostars #5472](https://neurostars.org/t/bids-validator-error-for-upper-case-subject-labels/5472):

> The bids specification allows for upper case labels for subjects:
> 
> BIDS uses custom user-defined labels in several situations (naming of participants, sessions, acquisition schemes, etc.) Labels are strings and MUST only consist of letters (lower or upper case) and/or numbers.
> 
> However when i use upper case e.g sub-X01, the bids validator (http://bids-standard.github.io/bids-validator/) says error 1 “NOT_INCLUDED” and if i use sub-x01 then it passes. I think in this regard, the bids validator is therefore inconsistent with bids specification…
> 
> Best wishes,
> 
> Ollie Hulme

**Comment by effigies** [comment]
Another user encountering this in [Neurostars #5496](https://neurostars.org/t/dont-understand-why-validation-fails-when-naming-scheme-matches/5496).

Anybody know how to debug javascript?

**Comment by sappelhoff** [comment]
I tried to replicate this on linux Ubuntu 18.04 with `master` validator and the `eeg_matchingpennies` example dataset.

1. Validate using `bids-validator eeg_matchingpennies/` --> all fine except for empty files
1. Rename `sub-05` in the dataset to `sub-X01`
1. Validate --> same as step 1, ... no additional errors

furthermore given the status in 

- https://neurostars.org/t/bids-validator-error-for-upper-case-subject-labels/5472 and
- https://neurostars.org/t/dont-understand-why-validation-fails-when-naming-scheme-matches/5496

I assume that this problem has been solved.
